encoder/vaapiencoder_base.cpp

```diff
@@ -568,6 +568,7 @@ YamiStatus VaapiEncoderBase::getOutput(VideoEncOutputBuffer* outBuffer, bool wit
     if (ret != YAMI_SUCCESS)
         return ret;
 
+    outBuffer->timeStamp = picture->m_timeStamp;
```

Comment:
This considered before, but getOutput is virtual function, h264 and h265(maybe more in future) have their own implemention. Better write it here because this only need one line.